sigopt_sklearn/search.py

```diff
@@ -257,7 +273,8 @@ def _fit(self, X, y, parameter_iterable=None):
         pre_dispatch = self.pre_dispatch
 
         # setup SigOpt experiment and run optimization
-        conn = sigopt.Connection(client_token=self.client_token)
+        conn = (self.sigopt_connection if self.sigopt_connection
```

Comment:
Sounds good. I wasn't sure if the `sigopt.Connection` constructor had any side-effects, so I held off on that. Will update PR.